checklists/dynamicmap.php

```diff
@@ -174,7 +295,7 @@ function checkForm(){
 						<div>
 							<label for="radius"><?php echo (isset($LANG['RADIUS'])?$LANG['RADIUS']:'Radius'); ?>:</label>
```

Comment:
Ideally, the `<input>` and `<select>` elements should have `<label>` elements accompanying them.